code/modules/antagonists/clockcult/clock_structures/traps/brass_skewer.dm

```diff
@@ -106,7 +106,7 @@
 		"<span class='danger'>You start tenderly lifting [skewee] off of [src]...</span>")
 		if(!do_after(user, 60, target = skewee))
 			skewee.visible_message("<span class='warning'>[skewee] painfully slides back down [src].</span>")
-			skewee.emote("moan")
+			skewee.forcesay("Oof, ouch owwie!!")
```

Comment:
`skewee` is a living mob arg of this proc, `forcesay()` is a human proc, mostly used to force people to blurt out what they were typing followed by an appe-GLORF!

```suggestion
			skewee.say("Oof, ouch owwie!!", forced = "fail brass skewer removal")
```

Or cut-paste forcesay() proc down to living/say.dm and make it a living proc ez pz.